### PR TITLE
Adds typings

### PR DESCRIPTION
**Pull request by aMarCruz**

### CODE REVIEW

**Comment by santomegonzalo** [comment]
Thank you @aMarCruz I will take a look and test it today :)

File: index.d.ts [hunk 1]
```diff
@@ -48,8 +48,8 @@ declare module "react-native-floating-action" {
     onPressMain?: (state: boolean) => any;
 
     // undocummented in the readme
-    actionsTextBackground: string;
-    actionsTextColor: string;
+    actionsTextBackground?: string;
```
Comment (santomegonzalo) [inline index.d.ts]: thanks for this... I completely forgot about this... I'm adding into the README and changing the name for `textBackground` and `textColor`

**Comment by santomegonzalo** [comment]
@aMarCruz I released a new version with this fix... we will still support the old actions but now we change the name for the future... could you update your file?

https://github.com/santomegonzalo/react-native-floating-action/releases/tag/1.9.0

**Comment by aMarCruz** [comment]
@santomegonzalo done.

Also, I was confused by some discrepancies between the readme, the proptypes, and the code.

In FloatingAction.js...
- propTypes has `buttonColor` in actions, instead `color`.

In the readme:
- `textColor`/`textBackground` should be in the main button table, since they are not properties of the actions (although they are intended for them).
- `position` (type number) is missed in the table for actions.

Regarding `position`, this property in button and actions has different type and purpose. Perhaps `order` is better for actions.

Thanks.

**Comment by aMarCruz** [comment]
@santomegonzalo sorry, package.json and yarn.lock was altered by my commit, can you copy and merge the index.d.ts alone? or should I make another commit.

**Pull request closed without merging**